bodybank/templates/base.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
nice, what do you want to use to make it look nice? just bootstrap? that's easiest to set up I think, specially with this plugin: https://django-bootstrap-v5.readthedocs.io/en/latest/